src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -98,9 +99,18 @@ private IEnumerable<CompletionItem> GetDeclarationCompletions(BicepCompletionCon
   name: $3
   $0
 }", context.ReplacementRange);
+
+                yield return CreateKeywordCompletion(LanguageConstants.TargetScopeKeyword, "Target Scope keyword", context.ReplacementRange);
             }
         }
 
+        private IEnumerable<CompletionItem> GetTargetScopeCompletions(SemanticModel model, BicepCompletionContext context)
```

Comment:
@anthony-c-martin I'm accessing the declared type in a kind of roundabout way so when we make scopes more extensible, we just have to touch the `DeclaredTypeManager`.